scripts/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_model.py

```diff
@@ -256,6 +263,19 @@ def fit_statuses(self, fit_statuses: list) -> None:
 
         self._fit_statuses = fit_statuses
 
+    @property
+    def fit_statuses_cache(self) -> list:
```

Comment:
This might be something to consider when I create the model analysis context next release. At the moment I think it is best kept here because it will be common code used across fitting, corrections and model analysis. If I move it to the context, I might need to create three different accessor methods for each of these tabs, which wouldn't be ideal

Reply:
dont we have a general fitting context that it could be stored in? 

Im thinking further down the line we can just dump the context for saving.

Reply:
Ah yes project save, I'll add these contexts as part of the model analysis epic #31445